host.go

```diff
@@ -24,6 +26,11 @@ var (
 	ErrInvalidHostname   = errors.New("Invalid hostname specified")
 )
 
+const (
+	swarmDockerImage              = "ehazlett/swarm:latest"
```

Comment:
There isn't an official yet.  It's coming.  I could make it an option as well -- would be good for testing.